packages/contentful/src/crn/queries/news.queries.ts

```diff
@@ -0,0 +1,47 @@
+import { gql } from 'graphql-tag';
+
+export const newsContentQueryFragment = gql`
+  fragment NewsContent on NewsCollection {
+    items {
+      sys {
+        firstPublishedAt
+      }
+      id
+      title
+      shortText
+      frequency
+      link
+      linkText
+      thumbnail {
+        url
+      }
+      text {
+        json
+      }
+    }
+  }
+`;
+
+export const FETCH_NEWS_BY_ID = gql`
+  query FetchNewsById($id: String!) {
+    newsCollection(where: { id: $id }) {
```

Comment:
why exactly are you using the collection when we could just use the `news` query which has an ID param?
![image](https://user-images.githubusercontent.com/11645956/200344767-a1fc626a-ceea-4c15-a8c1-bf3d30d3b429.png)

Reply:
This id is the contentful id. If we want to fetch by squidex id that we will migrate and populate this id field we need to query using the collection to filter properly